### PR TITLE
[Google Drive] Update display annotations in client endpoints

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
## Description
This PR updates display annotations according to the guidelines

### CODE REVIEW

File: drive/client.bal [hunk 1]
```diff
@@ -449,9 +449,9 @@ public client class Client {
     # 
     # + fields - The paths of the fields you want included in the response
     # + return - If successful, returns `About`. Else returns `error`
-    @display {label: "Get information about drive"} 
+    @display {label: "Get Information About Drive"}
```
Comment (abeykoon) [inline drive/client.bal:452]: Get Drive Information?

File: drive/client.bal [hunk 2]
```diff
@@ -462,12 +462,12 @@ public client class Client {
     # + pageToken - The page token  
     # + expiration - The expiration time
     # + return - If successful, returns `WatchResponse`. Else returns `error`
-    @display {label: "Watch specific using file id"} 
+    @display {label: "Watch Specific Using File Id"}
```
Comment (abeykoon) [inline drive/client.bal:465]: What is `WatchResponse `? would not `Get Watch Response` will work?

File: drive/client.bal [hunk 3]
```diff
@@ -69,7 +69,7 @@ public client class Client {
     # 
     # + optional - 'ListFilesOptional' used to add query parameters to the request
     # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
-    @display {label: "Get files"}
+    @display {label: "Get Files"}
     remote isolated function getFiles(@display {label: "Optional Query Parameters"} ListFilesOptional? optional = ())
```
Comment (abeykoon) [inline drive/client.bal:73]: Shall we name this as `Filtering Parameters`?

File: drive/client.bal [hunk 4]
```diff
@@ -83,7 +83,7 @@ public client class Client {
     # 
     # + filterString - The filter string
```
Comment (abeykoon) [inline drive/client.bal:84]: Shall we put google doc how this string should come?

File: drive/client.bal [hunk 5]
```diff
@@ -83,7 +83,7 @@ public client class Client {
...
     # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
-    @display {label: "Get all files"}
+    @display {label: "Get All Files"}
     remote isolated function getAllFiles(@display {label: "Filter String"} string? filterString = ())
```
Comment (abeykoon) [inline drive/client.bal:87]: Shall we merge the two operations  `getAllFiles ` and `filterFiles `. We can keep `getAllFiles` with orderBy as an optional. In Bal Doc specify we can receive all files filtered and optionally ordered.

File: drive/client.bal [hunk 6]
```diff
@@ -208,7 +208,7 @@ public client class Client {
     #             'modifiedTime', 'name', 'name_natural', 'quotaBytesUsed', 'recency', 'sharedWithMeTime', 'starred', 
     #              and 'viewedByMeTime'
     # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
-    @display {label: "Get forms by name"}
+    @display {label: "Get Forms By Name"}
```
Comment (abeykoon) [inline drive/client.bal:211]: Shall we say `Get Forms` ? The input parameter `Name` is obvious when he loads the input form.

File: drive/client.bal (hunk 1 above)
Comment (SkNuwanTissera) [reply drive/client.bal]: This function doesn't have much use case when it comes to low code. So removed the function.

File: drive/client.bal (hunk 2 above)
Comment (SkNuwanTissera) [reply drive/client.bal:425]: `WatchResponse` is the response we receive after subscribing to a watch. Currently, we have hidden these watch methods from choreo. By the way, we will send another PR after moving these methods to the listener.

File: drive/client.bal (hunk 3 above)
Comment (SkNuwanTissera) [reply drive/client.bal]: Now merge all the functionality to `getAllfiles`. Removed this function.

File: drive/client.bal (hunk 4 above)
Comment (SkNuwanTissera) [reply drive/client.bal]: Updated with link to readme

File: drive/client.bal (hunk 5 above)
Comment (SkNuwanTissera) [reply drive/client.bal]: Hi,
Did the changes. 
Thanks.

File: drive/client.bal (hunk 6 above)
Comment (SkNuwanTissera) [reply drive/client.bal]: Hi, 
Updated in all the places.
Thanks

**Review by abeykoon — APPROVED** [review]

**Pull request merged**